onnxscript/__init__.py

```diff
@@ -23,6 +23,24 @@
 except importlib_metadata.PackageNotFoundError:
     __version__ = None  # type: ignore[assignment]
 
+from .onnx_types import (
+    FLOAT,
```

Comment:
Looks like isort is wanting to sort alphabetically. We can use `isort: off` and `isort: on` around this block to preserve semantic ordering

https://pycqa.github.io/isort/docs/configuration/action_comments.html